app/gui2/src/components/lexical/theme.css

```diff
@@ -0,0 +1,48 @@
+.h1 {
```

Comment:
Let's add a comment how those classes are being used (link to `lexicalTheme` function).